capstone/src/main/webapp/youtubeScript.js

```diff
@@ -174,3 +198,75 @@ function isLetter(character) {
     return false;
   }
 }
+
+async function getTrending(categoryId) {
+  trendingResponse = await fetch('/trending_servlet?videoCategoryId=' + categoryId,)
+  trendingResponseJson = await trendingResponse.json();
```

Comment:
I think we should declare these with `const`, unless it produces an error I'm not aware of.

Reply:
addressed in "spacing and address Daniel's nits"